src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -0,0 +1,65 @@
+import { ModularTable } from "@canonical/react-components";
+import { useMemo } from "react";
+import { useParams } from "react-router-dom";
+import type { Column } from "react-table";
+
+import type { EntityDetailsRoute } from "components/Routes/Routes";
+import { generateFakeAuditLogs } from "pages/Logs/audit-logs-fake-data";
+
+const AuditLogsTable = () => {
+  const { modelName } = useParams<EntityDetailsRoute>();
+
+  const emptyMsg = `There are no audit logs available yet for ${modelName}`;
+  const columnData: Column[] = useMemo(
+    () => [
+      {
+        Header: "user tag",
+        accessor: "userTag",
+      },
+      {
+        Header: "model",
+        accessor: "model",
+      },
+      {
+        Header: "time",
+        accessor: "time",
+      },
+      {
+        Header: "facade name",
+        accessor: "facadeName",
+      },
+      {
+        Header: "facade method",
+        accessor: "facadeMethod",
+      },
+      {
+        Header: "facade version",
+        accessor: "facadeVersion",
+      },
+    ],
+    []
+  );
+  const fakeTableData = useMemo(() => {
+    const auditLogsData = generateFakeAuditLogs();
+    const tableData = auditLogsData.map((auditLogsEntry) => {
+      const { model } = auditLogsEntry;
+      const time = new Date(auditLogsEntry.time).toLocaleString();
```

Comment:
I think it would be nice to display this the same as it is done in the action log table with a relative date and the full date in a tooltip: https://github.com/canonical/juju-dashboard/blob/main/src/pages/EntityDetails/Model/Logs/ActionLogs/ActionLogs.tsx#L317-L322

<img width="272" alt="Screenshot 2023-07-26 at 11 41 44 am" src="https://github.com/canonical/juju-dashboard/assets/361637/4039874f-c24a-4e95-9e55-57c263db0d97">

Reply:
Done.